options/middleware.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
```

Comment:
The most appropriate is 'options' but this name is busy and deprecated.
What can you suggest?

Anyway we can declare different options in different files.